engine/lib/concrete_ident/concrete_ident.mli

```diff
@@ -29,6 +29,7 @@ val to_debug_string : t -> string
 module Create : sig
   val fresh_module : from:t list -> t
   val move_under : new_parent:t -> t -> t
+  val ctor : t -> t
```

Comment:
Can you add a comment to explain that this adds a `Ctor` to a concrete ident, and that this is useful to craft a constructor from a variant name?

Reply:
Let's name that `constructor` instead of `ctor` btw